src/inverseRobot.js

```diff
@@ -4,10 +4,34 @@
  * @param {object} robot
  *
  * @return {object}
-*/
-
+ */
 function inverseRobot(robot) {
-  // write code here
+  const keys = Object.values(robot);
+  const obj = {};
```

Comment:
Consider using more descriptive names for the variables(for example valueCounts instead of obj). Same for the rows below